recipes/ipython-sql/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3.6" %}
+{% set md5 = "d4feb00ac5806d7640b2545a43974766" %}
+
+package:
+  name: ipython-sql
+  version: {{ version }}
+
+source:
+  fn: ipython-sql-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython-sql/ipython-sql-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
```

Comment:
Well, if you want to just do Python 3.x, please add `skip: true  # [py2k]`. If we need Python 3.5 only, we may need to change `py2k` to `py35`. Once this is done, please drop the changes to `.CI/build_all` from the PR.